src/test/python/test_lists.py

```diff
@@ -88,7 +89,7 @@ def test_setslice(self):
     def test_add(self):
         jlist = makeJavaList()
         x = jlist + [1, 2, 3]
-        self.assertIn('jep.PyJList', str(type(x)))
+        self.assertTrue(isinstance(x, List.__pytype__))
```

Comment:
What happens if you replace `List.__pytype__` with `ArrayList`?

Reply:
ArrayList is a pyjclass, so it would throw an error. ArrayList.__pytype__ would be a type so that would work.